assembly/src/main/resources/template.vm

```diff
@@ -43,12 +43,12 @@ under the License.
         <script src='http://struts.apache.org/highlighter/js/$hscript' type='text/javascript'></script>
     #end
 
-    <script type="text/javascript">
+    <script>
         SyntaxHighlighter.defaults['toolbar'] = false;
         SyntaxHighlighter.all();
     </script>
 #end
-    <script type="text/javascript" language="javascript">
+    <script language="javascript">
```

Comment:
```suggestion
    <script>
```

Should the language attribute also be removed?

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#deprecated_attributes states:

>Like the type attribute, this attribute identifies the scripting language in use. Unlike the type attribute, however, this attribute's possible values were never standardized. The type attribute should be used instead.

Reply:
Good catch, thanks!